src/client/components/PressPackPage/index.js

```diff
@@ -0,0 +1 @@
+export { default, PressPack } from './PressPack';
```

Comment:
Teraz tu ci się wykrzaczy, bo zmieniłeś nazwę pliku na PressPackPage.js

Reply:
Już to edytowałe, chyba nie zapisałem przed wrzuceniem :D